aqueduct_client/examples/Sample Aqueduct Usage.ipynb

```diff
@@ -0,0 +1,533 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantopian Enterprise - Sample Aqueduct Usage\n",
+    "***\n",
+    "Aqueduct allows for programmatic access to Quantopian Enterprise models by providing REST endpoints to directly access the fast Quantopian calculation engine for factors.\n",
+    "\n",
+    "Models developed in Quantopian Enterprise can be run via RESTful requests to the Aqueduct server without any modification, streamlining the path from research to production.\n",
+    "\n",
+    "For more information on Aqueduct, including installation, please visit the [Setup](https://factset.quantopian.com/docs/user-guide/tools/aqueduct#setup) in Quantopian Enterprise."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import time\n",
+    "\n",
+    "from aqueduct_client import create_client\n",
+    "from aqueduct_client.utils import load_api_key"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running the AqueductClient\n",
+    "\n",
+    "Once the `aqueduct_client` and api key have been loaded, we can create an instance of client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "API_KEY = load_api_key()\n",
+    "client = create_client(API_KEY)"
+   ]
```

Comment:
What do you think about adding a cell under this one that displays the version of the client? Something like:
```
print('Aqueduct client version: {}'.format(aqueduct_client.__version__))
```

It might make debugging issues a little easier down the road.